src/components/ui/authButton.css

```
@@ -0,0 +1,18 @@
+.logout-button {
+  background: none;
+  border: none;
+  cursor: pointer;
+  border-radius: 0.25rem;
+  padding: 0.5rem 0.5rem;
+  border: 1px solid var(--sub-color);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)